src/main/kotlin/kotlinx/kover/KoverPlugin.kt

```diff
@@ -278,7 +279,34 @@ class KoverPlugin : Plugin<Project> {
     }
 }
 
-private class IntellijErrorLogChecker(private val taskExtension: KoverTaskExtension) : Action<Task> {
+/*
+  To support parallel tests, both Coverage Engines work in append to data file mode.
```

Comment:
Hm, I'm not sure one can append to file in parallel without causing it to be a mess. Are you sure it won't be a problem?